dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/language/Path.scala

```diff
@@ -25,7 +26,7 @@ object Path {
   // TODO replace with dynamic rendering based on the terminal's width, e.g. in scala-repl-pp
   lazy val maxTrackedWidth = sys.env.get("JOERN_DATAFLOW_TRACKED_WIDTH").map(_.toInt).getOrElse(DefaultMaxTrackedWidth)
```

Comment:
the comment above says we should get rid of this in a PR like this one here.

Reply:
True that. This is used to shorten the tracked statement (see a few lines below). Given that our tables now support cells with arbitrary length (which is being wrapped into multiple lines), we can probably just drop this altogether... 

Would you agree? 
And would you mind giving this a quick poweruser sanity test, once https://github.com/ShiftLeftSecurity/codepropertygraph/pull/1754 is merged? I'll let you know.